www/src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -21,101 +21,154 @@ import StickyResponsiveSidebar from 'components/StickyResponsiveSidebar';
 import TitleAndMetaTags from 'components/TitleAndMetaTags';
 import findSectionForPath from 'utils/findSectionForPath';
 import toCommaSeparatedList from 'utils/toCommaSeparatedList';
-import {sharedStyles} from 'theme';
+import {media, sharedStyles} from 'theme';
 import {urlRoot} from 'constants';
 
-const MarkdownPage = ({
-  authors,
-  date,
-  ogDescription,
-  location,
-  markdownRemark,
-  sectionList,
-  titlePostfix = '',
-}) => {
-  const hasAuthors = authors.length > 0;
-  const titlePrefix = markdownRemark.frontmatter.title || '';
-
-  return (
-    <Flex
-      direction="column"
-      grow="1"
-      shrink="0"
-      halign="stretch"
-      css={{
-        width: '100%',
-        flex: '1 0 auto',
-        position: 'relative',
-        zIndex: 0,
-        overflow: 'auto', // This is required for the mobile footer layout
-      }}>
-      <TitleAndMetaTags
-        title={`${titlePrefix}${titlePostfix}`}
-        ogUrl={`${urlRoot}/${markdownRemark.fields.path || ''}`}
-        ogDescription={ogDescription}
-      />
-      <div css={{flex: '1 0 auto'}}>
-        <Container>
-          <div css={sharedStyles.articleLayout.container}>
-            <Flex type="article" direction="column" grow="1" halign="stretch">
-              <MarkdownHeader title={titlePrefix} />
-
-              {(date || hasAuthors) &&
-                <div css={{marginTop: 15}}>
-                  {date}{' '}
-                  {hasAuthors &&
-                    <span>
-                      by {toCommaSeparatedList(authors, author => (
-                        <a
-                          css={sharedStyles.link}
-                          href={author.frontmatter.url}
-                          key={author.frontmatter.name}>
-                          {author.frontmatter.name}
-                        </a>
-                      ))}
-                    </span>}
-                </div>}
-
-              <div css={sharedStyles.articleLayout.content}>
-                <div
-                  css={[sharedStyles.markdown]}
-                  dangerouslySetInnerHTML={{__html: markdownRemark.html}}
-                />
+class MarkdownPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.toggleMenuOverlay = this.toggleMenuOverlay.bind(this);
+    this.handleWindowResize = this.handleWindowResize.bind(this);
+    this.state = {
+      isMenuOverlayOpen: false,
+    };
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (this.state.isMenuOverlayOpen && !prevState.isMenuOverlayOpen) {
+      // Menu has been opened
+      window.addEventListener('resize', this.handleWindowResize);
+    }
```

Comment:
Should probably clean-up on close too:
```js
if (this.state.isMenuOverlayOpen && !prevState.isMenuOverlayOpen) {
  // add handler
} else if (!this.state.isMenuOverlayOpen && prevState.isMenuOverlayOpen) {
  // remove handler
}
```
B'c not all closes will reload the page (eg Tutorial section). Or we could just get rid of the previous check entirely:
```js
if (this.state.isMenuOverlayOpen) {
  // add handler
} else {
  // remove handler
}
```